libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -462,6 +622,8 @@ void WaveClip::WriteXML(XMLWriter &xmlFile) const
    xmlFile.WriteAttr(wxT("offset"), mSequenceOffset, 8);
    xmlFile.WriteAttr(wxT("trimLeft"), mTrimLeft, 8);
    xmlFile.WriteAttr(wxT("trimRight"), mTrimRight, 8);
+   xmlFile.WriteAttr(wxT("rawAudioTempo"), mRawAudioTempo.value_or(0.), 8);
+   xmlFile.WriteAttr(wxT("clipStretchRatio"), mClipStretchRatio, 8);
```

Comment:
Skipping ahead to one of the easier small commits:  Persistency looks correct.  I might this up while I had an excuse to change it, eliminating the repetition of string literals like "rawAudioTempo" and the rest, using named constants.

Take or leave that suggestion.